src/gui/servernotificationhandler.cpp

```diff
@@ -101,7 +101,7 @@ void ServerNotificationHandler::slotNotificationsReceived(const QJsonDocument &j
         // https://github.com/owncloud/notifications/blob/master/docs/ocs-endpoint-v1.md#deleting-a-notification-for-a-user
         ActivityLink al;
         al._label = tr("Dismiss");
-        al._link  = Utility::concatUrlPath(ai->account()->url(), notificationsPath + "/" + json.value("notification_id").toString()).toString();
+        al._link  = Utility::concatUrlPath(ai->account()->url(), notificationsPath + "/" + QString::number(json.value("notification_id").toDouble())).toString();
```

Comment:
No need to get the value again. It is already in a._id and is parsed as a int which makes more sense to me that a double :wink:  https://github.com/nextcloud/client/blob/8c080d8141f0e267725b6dafbb5fe9d0ccf0cfa5/src/gui/servernotificationhandler.cpp#L79